src/pybind/mgr/dashboard/frontend/e2e/cluster/mgr-modules.po.ts

```diff
@@ -1,5 +1,220 @@
+import { $$, browser, by, element } from 'protractor';
+import { Helper } from '../helper.po';
 import { PageHelper } from '../page-helper.po';
 
 export class ManagerModulesPageHelper extends PageHelper {
-  pages = { index: '/#/mgr-modules' };
+  pages = {
+    index: '/#/mgr-modules'
+  };
+
+  // NOTABLE ISSUES: .clear() does not work on most text boxes, therefore using sendKeys
+  // a Ctrl + 'a' BACK_SPACE is used.
+  // The need to click the module repeatedly in the table is to ensure
+  // that the values in the details tab updated. This fixed a bug I experienced.
+
+  editMgrModule(name, tuple) {
+    // Selects the Manager Module and then fills in the desired fields.
+    // Doesn't check/uncheck boxes because it is not reflected in the details table.
+    // DOES NOT WORK FOR ALL MGR MODULES, for example, Device health
+    this.navigateTo();
+    browser.wait(Helper.EC.elementToBeClickable(this.getTableCell(name)), Helper.TIMEOUT);
+    this.getTableCell(name).click();
+    element(by.cssContainingText('button', 'Edit')).click();
+
+    for (let i = 0, entry; (entry = tuple[i]); i++) {
```

Comment:
Since you are not using the `i` inside the `for`, you can simply do:
```suggestion
    for (let entry of tuple) {
```

Same applies for the remaining `for`